include/buffer_tree.h

```diff
@@ -169,7 +169,7 @@ class BufferTree {
    * we would like to be accesible to all the bufferControlBlocks
    */
   static uint32_t page_size;
-  static const uint32_t serial_update_size = sizeof(Node) + sizeof(Node);
+  static const uint serial_update_size = sizeof(Node_ID) + sizeof(Node_ID);
```

Comment:
don't use uint. it's not portable.

Reply:
uints are sneaky buggers. I'll get rid of it